bonfire/bonfire.py

```diff
@@ -630,17 +544,18 @@ def _list_namespaces(available, mine, output):
             for ns in namespaces:
                 data[ns.name] = {
                     "reserved": ns.reserved,
-                    "ready": ns.ready,
-                    "requester": ns.requester_name,
+                    "status": ns.status,
```

Comment:
FYI @skarekrow not sure if your iqe-tools checks this field in the JSON returned here from list namespaces ... you might need to alter the key name you're looking for when bonfire v3 is released

Reply:
We check the `expires_in` field, but we do add the `ns.ready` field to a dict, so will note that. Thanks!